<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pingWindow</class>
 <widget class="QDialog" name="pingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ping Ekle</string>
  </property>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>321</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Ping Listesi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>330</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add List</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>274</width>
     <height>316</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Ping byteSize</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSpinBox" name="spinBox_byte">
      <property name="maximum">
       <number>99999</number>
      </property>
      <property name="value">
       <number>56</number>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Ping interval (ms)</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="label_duration">
      <property name="text">
       <string>Duration (second)</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0">
     <widget class="QSpinBox" name="spinBox_duration">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QPushButton" name="pushButton_durationUnlimited">
      <property name="text">
       <string>infinit ping</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="17" column="0">
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="displayFormat">
       <string>d.MM.yyyy HH:mm</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="17" column="1">
     <widget class="QCheckBox" name="checkBox">
      <property name="text">
       <string>Tarih </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="checkBox_KillMod">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Kill Mod</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QSpinBox" name="spinBox_interval">
      <property name="minimum">
       <number>500</number>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QSpinBox" name="spinBox_timeout">
      <property name="maximum">
       <number>99999</number>
      </property>
      <property name="value">
       <number>300</number>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>time out (ms)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_durationUnlimited</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBox_duration</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_duration</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_durationUnlimited</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>dateTimeEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_KillMod</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_interval</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>163</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_KillMod</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBox_duration</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_KillMod</sender>
   <signal>toggled(bool)</signal>
   <receiver>pushButton_durationUnlimited</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_KillMod</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
